lib/solidus_bactracs/api/shipment_serializer.rb

```diff
@@ -15,6 +15,7 @@ def call(sguid: nil)
         user = @shipment.user
         rma_type = safe_rma_type
         rp_location = get_rp_location
+        df_part = get_df_part
```

Comment:
the `get_` prefix [is considered a bad pattern in rails](https://github.com/rubocop/ruby-style-guide#accessormutator-method-names).  Let's avoid it unless we argue for why to use it.  In this case none of these variables probably needed to be named this way.